### DISCUSSION TITLE
Change sidebar offcanvas breakpoints

### DISCUSSION THREAD

**Discussion by KatyPurry** [bootscore/bootscore#207] [Q&A] [answered]
Hello,
I'm starting with Bootstrap / Sass, it's a lot of information but thank you for your amazing work ! 

By default the sidebar's offcanvas is hidden at ≥768px (md), I want to change the behaviour and displaying from ≥992px (lg). 

In sidebar.php I changed 
`<div class="offcanvas-md offcanvas-end" tabindex="-1" id="sidebar" aria-labelledby="sidebarLabel">`
by
`<div class="offcanvas-lg offcanvas-end" tabindex="-1" id="sidebar" aria-labelledby="sidebarLabel">`

And removed `col-md4 / col-md-8` in `sidebar.php` and `index.php`

The sidebar is hidden at 800px for example but I don't have the "Open the sidebar" button to display it. It is only displayed at 768px or less.

Thank you !

**Comment by justinkruit** [comment, marked as answer]
You should change `d-md-none` to `d-lg-none` on this line: https://github.com/bootscore/bootscore/blob/01f72c2fbfc5d6ed9e38fd25dd55b58e3884f937/sidebar.php#L20

That is what controls the visibility of the button. 😄 
And welcome to Bootstrap and Sass!

**Comment by KatyPurry** [comment]
Hello,
Oh yes, I was just reading : https://getbootstrap.com/docs/5.0/utilities/display/#hiding-elements 

Thank you !

**Comment by KatyPurry** [comment]
Just another question, on lg, button appears after the content, do you know why ?

Im' sorry, just found : `order-lg-last`

Resolved !

**Comment by crftwrk** [comment]
This seems to be a good point to start the new discussions feature. So, I converted this issue to a discussion because it isn‘t a bug or something else.

Thank you!
